.github/workflows/codespell.yml

```diff
@@ -0,0 +1,23 @@
+# Codespell configuration is within .codespellrc
+---
+name: Codespell
+
+on:
+  push:
+    branches: [develop]
+  pull_request:
+    branches: [develop]
+
+permissions:
+  contents: read
+
+jobs:
+  codespell:
+    name: Check for spelling errors
+    runs-on: ubuntu-latest
```

Comment:
Generally its fine but we like to lock the machines we run on, this would be ubuntu-22.04

Reply:
fair enough -- changing